packages/react-native-editor/__device-tests__/helpers/caps.js

```diff
@@ -1,12 +1,10 @@
 const ios = {
 	browserName: '',
 	platformName: 'iOS',
-	platformVersion: '13.4',
-	deviceName: 'iPhone 11',
 	os: 'iOS',
 	deviceOrientation: 'portrait',
 	automationName: 'XCUITest',
-	appiumVersion: '1.16.0', // SauceLabs requires appiumVersion to be specified.
+	appiumVersion: '1.17.1', // Sauce Labs requires appiumVersion to be specified.
```

Comment:
`appium` version on package.json is `1.18.3`. Is it expected to be different to the one declared here?

Reply:
`1.18.3` was not yet supported by Sauce Labs with iOS simulators: https://saucelabs.com/rest/v1/info/platforms/appium and older appium versions locally had issues with Xcode 12: https://github.com/appium/appium/issues/13532. As soon as Sauce Labs supports `1.18.3` with iOS we can bump this.